usr.bin/gzip/gzip.1

```diff
@@ -189,7 +189,7 @@ Options on the command line will override anything in
 .Sh EXIT STATUS
 The
 .Nm
-utility exits 0 on success,
+utility exits 0 on success,
```

Comment:
How do I compare spaces? Git diff only reports a diff if I change only this to a space, but... what is it?

Reply:
A normal space in 0x20 (in both ascii and utf-8).

UTF-8 also includes a "non-breaking space", which is the character pair 0xC2A0.  Non-breaking spaces are supposed to look the same as regular spaces (hence why they look the same on your console).

If you look at the raw bytes, then you can see the difference:
```
git diff | xxd
```
(hopefully your console shows color; look for `c2a0` near the end of the output)